RELEASE_NOTES.rst

```diff
@@ -1,3 +1,68 @@
+coala-bears 0.8.0
+=================
+
+For this release, we have had 19 contributors from around the world
+contributing about 176 commits to just coala-bears in the past 9 weeks.
```

Comment:
nono, what i meant is that you used "about 176 commits". how acn you say "about" and then such a precise number?